examples/scripts/utils.py

```diff
@@ -52,6 +54,12 @@ def make_tar(
     with tarfile.open(tarball_path, tarball_write_mode) as f:
         for dir in source_dir:
             dir = Path(dir)
-            f.add(dir, arcname=dir.name)
+            if os.path.isdir(dir):
```

Comment:
The `Path()` objects have a builtin method for doing this check:

```suggestion
            if dir.is_dir():
```